blocks/fragment/fragment.js

```diff
@@ -0,0 +1,33 @@
+import {
+  decorateMain,
+  loadBlocks,
+} from '../../scripts/scripts.js';
+
+/**
+ * Loads a fragment.
+ * @param {string} path The path to the fragment
+ * @returns {HTMLElement} The root element of the fragment
+ */
+async function loadFragment(path) {
+  const resp = await fetch(`${path}.plain.html`);
+  if (resp.ok) {
+    const main = document.createElement('main');
+    main.innerHTML = await resp.text();
+    decorateMain(main);
+    await loadBlocks(main);
+    return main;
+  }
+  return null;
+}
```

Comment:
We could discuss moving this function to `scripts.js` as it could also be useful in other scenarios (e.g. modals). But we can also wait until we get there...